### PR TITLE
feat(sms-limiting): db dependencies

### PR DESCRIPTION
**Pull request by seaerchin**
## Description
This PR adds a new key (`isOnboardedAccount`) to the database as well as a new (and unused) endpoint. This PR is a dependency of `sms-limiting`. 

This PR goes out 1 week in advance so that the db key can be created ahead of time. The `smscounts` database will then have all previous entries updated through running a db script (included in future PR) the night before the feature release. 

**Breaking Changes** 

- [ ] Yes - this PR contains breaking changes
    - Details ...
- [x] No - this PR is backwards compatible

### CODE REVIEW

File: src/app/modules/form/admin-form/admin-form.controller.ts [hunk 1]
```diff
@@ -2465,3 +2466,47 @@ export const handleUpdateStartPage = [
...
+  // Step 1: Check that the form exists
+  return (
+    FormService.retrieveFormById(formId)
+      // Step 2: Retrieve the free sms count
+      .andThen(({ admin }) => {
+        return SmsService.retrieveFreeSmsCounts(String(admin))
+      })
+      // Step 3: Map/MapErr accordingly
+      .map((freeSmsCountForAdmin) =>
+        res.status(StatusCodes.OK).json(freeSmsCountForAdmin),
```
Comment (karrui) [inline src/app/modules/form/admin-form/admin-form.controller.ts:2500]: Changes seem to be reverted here as compared to #2500

**Review by karrui — APPROVED** [review]
check that cherry pick actually worked?

File: src/app/modules/form/admin-form/admin-form.controller.ts (hunk 1 above)
Comment (seaerchin) [reply src/app/modules/form/admin-form/admin-form.controller.ts:2500]: yeap, #2500 got merged into `feat/sms-limiting` cos i didn't want to tack on more stuff here.

Comment (seaerchin) [reply src/app/modules/form/admin-form/admin-form.controller.ts]: cherry picked so #2500 is here; added a missing dependency in

**Pull request merged**